model/event/admin_event.go

```diff
@@ -39,7 +39,7 @@ type rawConfigDataChange struct {
 
 type rawAdminEventData struct {
 	GUID         string              `bson:"guid"`
-	ResourceType string              `bson:"r_type"`
+	ResourceType string              `bson:"r_type,omitempty"`
```

Comment:
Don't think we can omitempty here - there are queries checking for "". The migration will be changing the corresponding field to be empty, but until the field is removed I think we need to keep it here